src/functions-and-arrays.js

```diff
@@ -26,7 +42,13 @@ function sum() {}
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
-function averageNumbers() {}
+function averageNumbers(arrOfNum) {
+  let sum = arrOfNum[0];
+  for (let num of arrOfNum) {
+    sum += num
+  }
+  return sum / arrOfNum.length;
+}
```

Comment:
Same argument as above. You can basically use the sum function from above here. The function should also return null if the array is empty.

```
function averageNumbers(numbers) {
  if (numbers.length === 0) {
    return null;
  }
  const average = sumNumbers(numbers) / numbers.length;
  return average;
}
```